CHANGELOG.md

```diff
@@ -11,6 +11,9 @@
 - Minor: The points gain rate information at the top of the points page is now dynamically updated based upon your settings for the "Chatters Refresh" module.
 - Minor: "dev" config flag is now respected in web, properly omitting any git information in its footer
 - Minor: The subscriber badge is now automatically downloaded on web application startup. Which version of the subscriber badge should be downloaded can be configured in config.ini under the `web` section using the `subscriber_badge_version` key. Setting the `subscriber_badge_version` key to `-1` disables the sub badge downloading, in case you want to use a custom subscriber badge (or an old one that you don't want to overwrite)
+- Minor: Update jQuery from 2.1.4 to 3.4.1.
+- Minor: Fix playsounds not working after updating jQuery.
```

Comment:
`Minor: Fix playsounds not working after updating jQuery.` should not be in the changelog, since it's not really behaviour that has changed since the latest version

Reply:
I would also say the jquery update does not have to be in the changelog, since it's not a change that affects the bot users or the bot admin/operator at all

Reply:
I removed the 2 entries. It was just me being me with documenting every change made.